### PR TITLE
Unified buttons/ UI and new screen to Update user posts

### PR DESCRIPTION
**Pull request by Asdesai99**
Updated all buttons to follow a similar theme, and updated some UI to be more consistent. 

**Added a new screen, EditPostScreen**
- accessible through clicking on a post on user profile screen

### CODE REVIEW

**Comment by Asdesai99** [comment]
for the LoginScreen accept my style sheet, its more or less the same just a few changes

**Comment by ryanhow10** [comment]
Great work Akash, these changes really make our application look more professional! I was able to run the frontend project and view all the buttons and indeed all of their styles were the same. One nit pick improvement we could make is creating a style file where we defined regularly used styles and that way we won't need to copy and paste styles in multiple portions of the code. As for the updating user posts screen, I was able to successfully update a post from that screen and all updated fields were updated in the database. Everything seems good to me.

File: screens/EditPostScreen.tsx [hunk 1]
```diff
@@ -0,0 +1,280 @@
...
+        post_id: post.post_id,
+        title: post.title,
+        requesting: post.requesting,
+        content: post.content,
+        tags: post.tags,
+    });
+
+    const updatePost = () => {
+        axios
+            .put(`http://localhost:3000/api/v1/posts/${post.post_id}`, updatedPost)
```
Comment (ryanhow10) [inline screens/EditPostScreen.tsx:30]: We should probably use an environment variable instead of hardcoding

File: screens/EditPostScreen.tsx [hunk 2]
```diff
@@ -0,0 +1,280 @@
...
+            .then(resp => {
+                setPostToUpdate(resp.data)
+            })
+            .catch(err => {
+                console.log(err);
```
Comment (ryanhow10) [inline screens/EditPostScreen.tsx:35]: Maybe in the future we could implement a success or error message on update to notify the user of the state of the post

File: screens/EditPostScreen.tsx (hunk 1 above)
Comment (Asdesai99) [reply screens/EditPostScreen.tsx:30]: Yes when we deploy we will change it, but for now using localhost is much faster for development purposes

File: screens/EditPostScreen.tsx (hunk 2 above)
Comment (Asdesai99) [reply screens/EditPostScreen.tsx:35]: hmm great idea I will look into it.

**Comment by alanwu4321** [comment]
Great work Akash! I have some NITs maybe we could include in the next iteration
- Follow conventional Git commit message https://www.conventionalcommits.org/en/v1.0.0/
- Include screenshots and better explanations in the PR description so the person approving and merging your PR can have all the context necessary at a glance. This would improve the agility and efficiency of team development as well.
- The button could be imported from `react-native-paper` instead of `react-native`. `react-native-paper` allows a more robust and powerful API that supports icon, loading, ..etc. https://callstack.github.io/react-native-paper/button.html for more

**Review by 17yongga — APPROVED** [review]
Good work on the user profile! Merging to main for deployment

**Pull request merged**